config/cloudinary.yml

```diff
@@ -0,0 +1,21 @@
+---
+development:
+  cloud_name: db3kljtw9
+  api_key: "932935994286465"
+  api_secret: wyze4QdaUwsDzzXNjXkiClskBbs
+  enhance_image_tag: true
+  static_file_support: false
+
+production:
+  cloud_name: db3kljtw9
+  api_key: "932935994286465"
+  api_secret: wyze4QdaUwsDzzXNjXkiClskBbs
+  enhance_image_tag: true
+  static_file_support: true
+
+test:
+  cloud_name: db3kljtw9
+  api_key: "932935994286465"
+  api_secret: wyze4QdaUwsDzzXNjXkiClskBbs
```

Comment:
You should try to add these to env vars or secrets, it's not very secure to have them in the repository.

Also, if you are not updating the `active_storage.service` in the `environments/test.rb` file, the app will not use Cloudinary in the test environment, so you could remove the `test` credentials